### ISSUE TITLE
/tests/ffi/agent-flood is failing

### ISSUE THREAD

**Issue by Yarboa** [containers/qm#517]
During the work to reduce download time of container ffi-tools

The tests has started to fail.

```
bluechictl status
NODE                          | STATE     | IP                      | LAST SEEN                   
==========================================================================================
qm.localrootfs                | online    | ::ffff:10.0.2.15        | now                         
localrootfs                   | offline   |                         | never                       
bluechi-tester-1              | offline   |                         | never                       
bluechi-tester-2              | offline   |                         | never 
```

Found in issue #511 #510

**Comment by pengshanyu** [comment]
I ran agent-flood against local VM (provision via the latest c9s cloud image) and encountered another issue:
        ```
+ info_message 'Starting setup'
        + '[' -z 'Starting setup' ']'
        + BLUE='\033[94m'
        + ENDCOLOR='\033[0m'
        + echo -e '[ \033[94mINFO\033[0m  ] Starting setup'
        + info_message ==============================
        + '[' -z ============================== ']'
        + BLUE='\033[94m'
        + ENDCOLOR='\033[0m'
        + echo -e '[ \033[94mINFO\033[0m  ] =============================='
        + '[' 0 -ne 0 ']'
        + echo
        + info_message 'Checking if QM already installed'
        + '[' -z 'Checking if QM already installed' ']'
        + BLUE='\033[94m'
        + ENDCOLOR='\033[0m'
        + echo -e '[ \033[94mINFO\033[0m  ] Checking if QM already installed'
        + info_message ==============================
        + '[' -z ============================== ']'
        + BLUE='\033[94m'
        + ENDCOLOR='\033[0m'
        + echo -e '[ \033[94mINFO\033[0m  ] =============================='
        ++ systemctl is-enabled qm
        + QM_STATUS='Failed to get unit file state for qm.service: No such file or directory'
```
and I checked the test log https://artifacts.dev.testing-farm.io/ebc86564-f869-487f-93f9-57f58c3631ea/work-ffifknu8_or/log.txt for #508 and found:
```
16:30:36             err: + echo -e '[ \033[94mINFO\033[0m  ] Checking if QM already installed'
16:30:36             err: + info_message ==============================
16:30:36             err: + '[' -z ============================== ']'
16:30:36             err: + BLUE='\033[94m'
16:30:36             err: + ENDCOLOR='\033[0m'
16:30:36             err: + echo -e '[ \033[94mINFO\033[0m  ] =============================='
16:30:36             err: ++ systemctl is-enabled qm
16:30:36             err: + QM_STATUS=generated
16:30:36             err: + '[' generated == generated ']'
16:30:36             err: ++ systemctl is-active qm
16:30:36             err: + '[' inactive == active ']'
16:30:36             err: + test -d /var/qm -a -d /etc/qm
16:30:36             err: + stat /run/ostree-booted
16:30:36             err: + info_message 'Cleaning any previous e2e files'
```
You can see that the value of QM_STATUS  ​​in the above two logs are different. 
@Yarboa Could you please give me some help, am I missing something here?

**Comment by pengshanyu** [comment]
I saw that in the latest gate test, agent-flood passed https://artifacts.dev.testing-farm.io/e9a370f9-bcaa-4bd0-8707-4c2390d606d3/

I checked the previously failed tests and the failed information is `test() bluechi-tester-1 is not active`. Further observation is needed to confirm why `bluechi-tester-1` did not start successfully.

For the image where qm is already installed and the qm service is `active`, the `set-ffi-env-e2e` script is not executed, I need to confirm whether bluechi services is enabled and started there.
